app/src/main/kotlin/com/wire/android/ui/home/conversations/messages/ConversationMessagesViewModel.kt

```diff
@@ -118,11 +122,19 @@ class ConversationMessagesViewModel @Inject constructor(
     }
 
     private fun loadPaginatedMessages() = viewModelScope.launch {
-        val paginatedMessagesFlow = getMessageForConversation(conversationId)
+        val lastReadIndex = when (val result = getConversationUnreadEventsCount(conversationId)) {
+            is GetConversationUnreadEventsCountUseCase.Result.Success -> result.amount.toInt()
```

Comment:
why is it not int from Kalium ? 🤔

Reply:
cant we just work with Long ? otherwise I see not sense to return Long from Kalium 🤔

Reply:
Well, there is DB-request under the hood, it may fail. 
I believe it's better to return error and let client-side decide what to do on it, instead of returning some default value